pages/product-list.html

```diff
@@ -0,0 +1,448 @@
+
+                <li class="border-bottom">
+                  <a class="dropdown-item fw-medium py-5 px-3 text-neutral-700" href="#">植栽單品</a>
+                </li>
+
+                <li class="border-bottom">
+                  <a class="dropdown-item fw-medium py-5 px-3 text-neutral-700" href="#">療癒組盆</a>
+                </li>
+
+                <li class="border-bottom">
+                  <a class="dropdown-item fw-medium py-5 px-3 text-neutral-700" href="#">客製禮盒</a>
+                </li>
+
+                <li class="border-bottom py-5 px-3 accordion-item">
+                  <button
+                    type="button"
+                    class="list-group-item list-group-item-action text-neutral-700 fw-medium d-flex accordion-header"
+                    data-bs-toggle="collapse"
+                    data-bs-target="#collapseAccessories"
+                    aria-expanded="false"
+                  >
+                    配件商品
+                    <span class="material-symbols-rounded align-bottom ms-auto accordion-icon">
+                      keyboard_arrow_down
+                    </span>
+                  </button>
+
+                  <div class="collapse" id="collapseAccessories">
+                    <ul class="list-group list-unstyled">
+                      <li>
+                        <a
+                          href="#"
+                          class="list-group-item accessories-item border-0 pt-5 pb-0 px-0 fs-sm text-neutral-400"
+                          >土壤</a
+                        >
+                      </li>
+
+                      <li>
+                        <a
+                          href="#"
+                          class="list-group-item accessories-item border-0 pt-5 pb-0 px-0 fs-sm text-neutral-400"
+                          >盆器</a
+                        >
+                      </li>
+
+                      <li>
+                        <a
+                          href="#"
+                          class="list-group-item accessories-item border-0 pt-5 pb-0 px-0 fs-sm text-neutral-400"
+                          >裝飾物</a
+                        >
+                      </li>
+                    </ul>
+                  </div>
+                </li>
+              </ul>
+            </div>
+          </div>
+
+          <!-- 手機版toggle -->
+          <div class="col-6">
+            <div class="d-flex text-center justify-content-around py-2">
+              <a href="#" class="d-block w-50 fs-sm fw-medium text-primary">最新</a>
+              <a href="#" class="d-block w-50 fs-sm fw-medium text-primary-400 border-start">熱門</a>
+            </div>
+          </div>
+        </div>
+
+        <!-- 手機版products -->
+        <div class="row d-lg-none">
+          <div class="col-6 mb-6">
+            <%- include('./components/product-card',{ picSrc: "../assets/images/img_product_07.png", picAlt: "雪夜之森",
+            productTag: "植感精選", productName: "雪夜之森", productPrice: "NT$2,400", }); -%>
```

Comment:
```suggestion
            <%- include('./components/product-card',{ picSrc: "../assets/images/img_product_07.png", picAlt: "雪夜之森",
            productTag: "植感精選", productName: "雪夜之森", productPrice: "NT$2,400", }); -%>
```
請問 @wc-su 是指這邊的`productTag: "植感精選"`嗎?